### ISSUE TITLE
webpack alias not work.

### ISSUE THREAD

**Issue by liangxiwei** [electron-react-boilerplate/electron-react-boilerplate#3153]
## Summary

I want to use something like that in main process:
```import {xxx} from 'common/utils'```
my dir is :

src
---main
---renderer
---common

My webpack.config.base.ts is like that:
```
resolve: {
    extensions: ['.js', '.jsx', '.json', '.ts', '.tsx'],
    alias: {
      common: resolve(webpackPaths.srcPath, 'common'),
    },
    modules: [webpackPaths.srcPath, 'node_modules'],
  },
```
please tell me how can I config it?

**Comment by lobut** [comment]
I just randomly came across this, so I took a quick look.  It looks like if you do the change you requested on the `renderer` it seems to work, so I'm assuming it's on the `main.ts`.

The way that it currently works is that the `webpack.config.renderer.dev` will kick off the main build with *no* webpack on the server side: https://github.com/electron-react-boilerplate/electron-react-boilerplate/blob/main/.erb/configs/webpack.config.renderer.dev.ts#L166-L172. Now, that's only on the dev side.  For the production build, it will work as you expect, but I assume that you probably want to do a development build and not a production build.  In this case, you need to add your own webpack configuration to run as well.

I created a repo where I did a very simple example of this right here: https://github.com/lobut/electron-ss-webpack

If you're interested in only what's needed to get it basically working, you can check out the commit over here: https://github.com/lobut/electron-ss-webpack/commit/c752759a0c48894610cb8cb3f5a8301364892090.

I just copied the production webpack configuration and swapped to development.  Then I just run it before we run electron against the output.

This will not do any auto-reload or anything, you will need to shut down the process for any `main.ts` changes.  

This probably isn't the best example, but please let me know if it helps or have any further questions.

**Comment by liangxiwei** [comment]
> I just randomly came across this, so I took a quick look. It looks like if you do the change you requested on the `renderer` it seems to work, but it definitely doesn't work on the `main.ts` so I'm assuming we're just talking about the `main.ts` entrypoint.
> 
> The way that it currently works is that the `webpack.config.renderer.dev` will kick off the main build with _no_ webpack on the server side: https://github.com/electron-react-boilerplate/electron-react-boilerplate/blob/main/.erb/configs/webpack.config.renderer.dev.ts#L166-L172. Now, that's only on the dev side. For the production build, it will work as you expect, but I assume that you probably want to do a development build and not a production build. In this case, you need to add your own webpack configuration to run as well.
> 
> I created a repo where I did a very simple example of this right here: https://github.com/lobut/electron-ss-webpack
> 
> **TLDR** If you're interested in only what's needed to get it basically working, you can check out the commit over here: [lobut/electron-ss-webpack@c752759](https://github.com/lobut/electron-ss-webpack/commit/c752759a0c48894610cb8cb3f5a8301364892090).
> 
> I just copied the production webpack configuration and swapped to development. Then I just run it before we run electron against the output.
> 
> This will not do any auto-reload or anything, you will need to shut down the process for any `main.ts` changes.
> 
> This probably isn't the best example, but please let me know if it helps or have any further questions.

thank you for your commit.

I try your commmit and there has other problems...In the main process dev enviroment, it can't read assets directory file :(
